src/mpid/common/genq/mpidu_genq_shmem_queue.h

```diff
@@ -52,6 +52,30 @@ static inline MPIDU_genqi_shmem_cell_header_s
     for (void *tmp = MPIDU_genq_shmem_queue_head((queue)), cell = tmp; tmp; \
          tmp = MPIDU_genq_shmem_queue_next(tmp))
 
+static inline int MPIDU_genq_shmem_queue_init(MPIDU_genq_shmem_queue_t queue,
```

Comment:
Why is queue init moved this moved to `MPIDU_genq_shmem_pool.c`?

Reply:
invalid.